prometheus/testutil/promlint/promlint_test.go

```diff
@@ -499,6 +499,63 @@ x_bytes 10
 	runTests(t, tests)
 }
 
+func TestLintGauge(t *testing.T) {
+	tests := []test{
+		{
+			name: "counter with _timestamp_seconds suffix",
+			in: `
+# HELP _timestamp_seconds Test metric.
+# TYPE _timestamp_seconds counter
+_timestamp_seconds 10
+`,
+			problems: []promlint.Problem{
+				{
+					Metric: "_timestamp_seconds",
+					Text:   `counter metrics should have "_total" suffix`,
+				},
+				{
+					Metric: "_timestamp_seconds",
```

Comment:
This does not feel helpful for counters, the above problem duplicates here kind of and is kind of awkward why `timestamp_second` is not allowed. Can I do `ts_seconds`?